source/common/tls/context_impl.h

```diff
@@ -49,8 +49,8 @@ struct TlsContext {
   bssl::UniquePtr<X509> cert_chain_;
   std::string cert_chain_file_path_;
   std::unique_ptr<OcspResponseWrapper> ocsp_response_;
-  // ec_group_curve_name_ is initialized to zero, which is used as a sentinel value
```

Comment:
In the future, you can put words in a comment in backticks and the spell checker will ignore them. But it's fine to leave this as-is if you want.